pkg/cmd/cmd_run.go

```diff
@@ -43,53 +44,24 @@ func NewRunCmd() *cli.Command {
 			},
 		},
 		Action: func(c *cli.Context) error {
+			pkgWillBeCompiled := c.Args().First()
 
-			tarPath := c.String("tar")
-			compileResult := ""
-
-			// If a tar package specified by "--tar" to run
-			if len(tarPath) != 0 {
-				absTarPath, err := absTarPath(tarPath)
+			// 'kpm run' compile the current package undor '$pwd'.
+			if len(pkgWillBeCompiled) == 0 {
+				compileResult, err := runPkg(c.String("input"), c.Bool("vendor"))
```

Comment:
Use constants for flags `input` and `vendor`

Reply:
fixed.